app/src/main/kotlin/pl/gov/mc/protego/bluetooth/BeaconIdManager.kt

```diff
@@ -0,0 +1,68 @@
+package pl.gov.mc.protego.bluetooth
+
+import android.os.Handler
+import pl.gov.mc.protego.bluetooth.beacon.BeaconId
+import pl.gov.mc.protego.bluetooth.beacon.BeaconIdAgent
+import pl.gov.mc.protego.bluetooth.beacon.BeaconIdLocal
+import pl.gov.mc.protego.bluetooth.beacon.BeaconIdRemote
+import timber.log.Timber
+import java.util.*
+import java.util.concurrent.CopyOnWriteArraySet
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicInteger
+
+
+class BeaconIdManager : BeaconIdAgent {
```

Comment:
Może warto dodać jakiś krótki komentarz mówiący o tym, że ta klasa jest używana w środowisku wielowątkowym?

Reply:
Tak na prawdę to nie jest. Dodałem najprostszy sposób który powinien pomóc na najczęstszy przypadek. Nie widzę specjalnie potrzeby dla korzystania z tej klasy w środowisku wielowątkowym. Więcej – z racji na znane problemy z implementacją BLE na Androidzie raczej będzie wykorzystywana używając jednego wątku... Nie mniej dodam notkę.

Reply:
Dodałem `KDoc`a do klasy który powinien odpowiadać na Twoją prośbę